packages/linter/src/generators/init/init-migration.ts

```diff
@@ -38,9 +40,8 @@ export function migrateConfigToMonorepoStyle(
 export function findLintTarget(
   project: ProjectConfiguration
 ): TargetConfiguration {
-  return Object.entries(project.targets ?? {}).find(
-    ([name, target]) =>
-      name === 'lint' ||
```

Comment:
This is very prone to misuse and does not guarantee that the lint uses the ESLint config.